src/test/java/com/epam/tc/hw2/WebDriverElement.java

```diff
@@ -0,0 +1,26 @@
+package com.epam.tc.hw2;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import java.util.concurrent.TimeUnit;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+
+public class WebDriverElement {
```

Comment:
Please choose another name for the class, that provides information regarding its purpose

Reply:
done